xbmc/music/MusicUtils.h

```diff
@@ -80,12 +80,27 @@ std::vector<std::string> GetArtTypesToScan(const MediaType& mediaType);
   */
 bool IsValidArtType(const std::string& potentialArtType);
 
+/*! \brief Check whether auto play next item is set for the given item.
+  \param item [in] the item to check
+  \return True if auto play next item is active, false otherwise.
+  */
+bool IsAutoPlayNextItem(const CFileItem& item);
+
+enum class PlayMode
+{
+  CHECK_AUTO_PLAY_NEXT_ITEM, // play from here, or play only this, according to settings value
```

Comment:
Those comments are not recognized by doxygen afaik, maybe use `/*! my stuff */`

Reply:
Good catch. I will change that.